infrastructure/terraform/accounts/modules/account/aws/variables.tf

```
@@ -0,0 +1,10 @@
+variable "prefix" {}
+
+variable "principals" {
+  type = list
+}
+
+variable "max_session_duration_in_seconds" {
+  # Default is one hour
+  default = "3600"
```

As above, maybe calculate on the fly to make it clearer?

```suggestion
  default = 60 * 60
```

Also, you can set `type = number` here, which means you'll get better errors if you pass bad values here, and Terraform will auto-coerce strings into a number for you.